### PR TITLE
[to be merged with 2 other address search PRs] Address search

### PR DESCRIPTION
**Pull request by camillebaldock**

### CODE REVIEW

**Comment by russwillis** [comment]
All looks good.

**Comment by Pheonnexx** [comment]
reviewed this alongside the other stuff looks legit

File: service/server.py [hunk 1]
```diff
@@ -255,6 +256,13 @@ def get_register_titles_via_postcode(postcode):
     return results
 
 
+def get_register_titles_via_address(address):
+    response = requests.get(
```
Comment (lgonczar) [inline service/server.py:260]: It would be ideal to check if the response is a 200 response.

File: tests/test_app.py [hunk 2]
```diff
@@ -134,12 +138,13 @@ def test_title_search_success(self, mock_get):
         assert 'Luton' in page_content
         assert 'LU1 1DZ' in page_content
 
-    def test_title_search_invalid_search_value_format(self):
+    @mock.patch('requests.get', return_value=fake_no_titles)
```
Comment (lgonczar) [inline tests/test_app.py:141]: By analogy, we could check if the title appears in the search result if contained in the API response.

**Pull request merged**